apps/spruce/cypress/integration/spawn/host.ts

```diff
@@ -242,7 +242,25 @@ describe("Navigating to Spawn Host page", () => {
     cy.dataCy("edit-spawn-host-modal").should("be.visible");
 
     cy.getInputByLabel("Temporary Sleep Schedule Exemption").click();
-    cy.get("td[aria-current=true]").next().click();
+    cy.get("td[aria-current=true]").as("currentDateCell");
+    // Select a date in the future either this month or next month
+    // if the current date is the last day of the month select the first day of the next month
+    // if it is not select the next day
+    // We can determine if the current date is the last day of the month if the next cell is disabled
+    cy.get("@currentDateCell")
+      .next()
+      .then(($el) => {
+        if ($el.attr("aria-disabled") === "false") {
```

Comment:
I _think_ this would fail if the last day of the month fell on a Saturday; there would be no next `td` to grab. We can probably do this
```suggestion
        if (!$el || $el.attr("aria-disabled") === "false") {
```